### PR TITLE
"Insert a Wazuh API entry automatically" page removed and its content moved - for branch 3.12 

### LINKED ISSUE #2390: Review Insert a Wazuh API entry automatically page
Hello team!

The aim of this issue is to remove the information related to Kibana App on [this page](https://documentation.wazuh.com/3.12/installation-guide/installing-elastic-stack/automatic_api.html) and move the related to Splunk into the Splunk's sections, which is more appropriate.

Regards,

David

### PR DESCRIPTION
**Pull request by dkempny**
Hello Team,
this PR closes #2390 
 
## Description
For branch **3.12**
Installation guide -> Installing Elastic Stack -> **Insert a Wazuh API entry automatically** page is removed as it contained obsolete information about the Wazuh Kibana plugin.  All references to the page are removed. The information about Splunk app, previously presented on the page, is moved to the Installing Splunk -> Install Wazuh app for Splunk page.

**NOTE**: Removed page: source/installation-guide/installing-elastic-stack/automatic_api.rst

## Checks

- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns. 

All the best,
Daria

### CODE REVIEW

File: source/installation-guide/installing-splunk/splunk-app.rst [hunk 1]
```diff
@@ -44,7 +44,7 @@ Installation
     .. image:: ../../images/splunk-app/app-icon.png
       :align: center
 
-4. The app will redirect you to the *Settings* tab, where fill the form with your **Wazuh API credentials**. Use the URL and port from your Wazuh API server.
+4. The app will redirect you to the *Settings* tab, where fill the form with your ``Wazuh API credentials``. Use the URL and port from your Wazuh API server.
```
Comment (davidfmiranda) [inline source/installation-guide/installing-splunk/splunk-app.rst:47]:
```suggestion
#. The app will redirect you to the *Settings* tab, where fill the form with your ``Wazuh API credentials``. Use the URL and port from your Wazuh API server.
```

Comment (davidfmiranda) [reply source/installation-guide/installing-splunk/splunk-app.rst:47]: We should try to use always `#.` for enumerations.

File: source/installation-guide/installing-splunk/splunk-app.rst [hunk 2]
```diff
@@ -56,6 +56,18 @@ Installation
...
 
+    If you want to add the ``Wazuh API credentials`` more quickly (for instance, for deployment purposes) you can execute the following command:
+
+    .. code-block:: console
+
+      # curl -X POST "http://<SPLUNK_IP>:<SPLUNK_PORT>/en-US/custom/SplunkAppForWazuh/manager/add_api?url=<WAZUH_API_URL>&portapi=<WAZUH_API_PORT>&userapi=<WAZUH_API_USERNAME>&passapi=<WAZUH_API_PASSWORD>"
+
+    Note the following:
+
+    1. ``<SPLUNK_IP>`` is the **hostname or IP address** of the Splunk instance where the app was installed.
```
Comment (davidfmiranda) [inline source/installation-guide/installing-splunk/splunk-app.rst:67]:
```suggestion
    #. ``<SPLUNK_IP>`` is the **hostname or IP address** of the Splunk instance where the app was installed.
```

File: source/installation-guide/installing-splunk/splunk-app.rst [hunk 3]
```diff
@@ -56,6 +56,18 @@ Installation
...
+    2. ``<SPLUNK_PORT>`` is the **port of the Splunk instance** where the app was installed. By default, it's 8000.
```
Comment (davidfmiranda) [inline source/installation-guide/installing-splunk/splunk-app.rst:68]:
```suggestion
    #. ``<SPLUNK_PORT>`` is the **port of the Splunk instance** where the app was installed. By default, it is 8000.
```

File: source/installation-guide/installing-splunk/splunk-app.rst [hunk 4]
```diff
@@ -56,6 +56,18 @@ Installation
...
+    3. ``<WAZUH_API_URL>``, ``<WAZUH_API_PORT>``, ``<WAZUH_API_USERNAME>`` and ``<WAZUH_API_PASSWORD>`` represent the **Wazuh API credentials** to be stored on the app. Keep in mind that the Wazuh API URL must include ``http://`` or ``https://``, depending on the current configuration.
```
Comment (davidfmiranda) [inline source/installation-guide/installing-splunk/splunk-app.rst:69]:
```suggestion
    #. ``<WAZUH_API_URL>``, ``<WAZUH_API_PORT>``, ``<WAZUH_API_USERNAME>`` and ``<WAZUH_API_PASSWORD>`` represent the **Wazuh API credentials** to be stored on the app. Keep in mind that the Wazuh API URL must include ``http://`` or ``https://``, depending on the current configuration.
```

**Review by alberpilot — APPROVED** [review]

**Review by alberpilot — APPROVED** [review]

**Pull request merged**